handlers/desired_lrp_handlers.go

```diff
@@ -489,3 +505,32 @@ func (h *DesiredLRPHandler) updateInstances(ctx context.Context, logger lager.Lo
 		}
 	}
 }
+
+func (h *DesiredLRPHandler) logDesiredLrpParsingErrors(err *models.Error, processGuid string) error {
+	if err == nil {
+		return errors.New("parsing error is nil")
+	}
```

Comment:
**Issue**

With the way this function is invoked, `err` will never be `nil`. Also, what is the reasoning for returning an error even when there is no error?

Reply:
That's right, ill remove the check